assets/css/custom.css

```diff
@@ -85,16 +85,6 @@ body {
 	-webkit-tap-highlight-color: rgba(0,0,0,0);
 }
 
-/* Stops long links from breaking out of container, use on <a> tag */
-@media (max-width:400px) {
-	.dont-break-out {
-		overflow-wrap: break-word;
-		word-wrap: break-word;
-		-ms-word-break: break-all;
-		word-break: break-all;
-	}
-}
-
```

Comment:
ah yes sure, do you want me to move the link changes to a separate PR? 

Basically I decided it made more organizational sense to create link and secondary link style classes instead of having to apply the don't break out solution (which wasn't a perfect fit for our problem)